arbitrary_canton_sha

```diff
@@ -0,0 +1 @@
+6988cf6984a2a1cd3f720554adea5b839835217b
```

Comment:
I assume this shouldn't be committed?

Reply:
Right, it was needed while I was waiting for a second PR to canton to merge, but that merged this morning.
Sadly I cannot drop this however until canton pushes artefacts